### DISCUSSION TITLE
Realtime syncing not possible, so how often may I sync?

### DISCUSSION THREAD

**Discussion by Neobond** [arabcoders/watchstate#557] [General] [answered]
I saw the default library sync task was every 2 hours. If it is not possible to detect library changes and then automatically run the task, is there any downside to having the library sync more often, like every 5 minutes?

Or is that too taxing if it is checking the entire library state of each service each time it runs?

Right now I have it set to 30 mins.

Thanks for this awesome project!

**Comment by arabcoders** [comment, marked as answer]
Hi,

Its expensive to compute the changes in such short timer, and honestly wasteful of your resources. 

You can enable webhooks and leave the default regular sync time as it is. 

Webhooks will sync the state within 2-4min at most. usually 2min. 

I do not recommend lowering the import/export time if your media content library is large.

**Reply by Neobond** [reply]
Thanks! I did enable webhooks in Plex and Emby, so I will change it back.
